docs/getting-started.adoc

```diff
@@ -219,3 +219,70 @@ To list all enviroment variables in the current local configuration:
 ----
 odo config view
 ----
+
+== Using debug mode to interactively debug (Tech Preview)
+
+Odo allows the user to attach a debugger for remotely debugging your application. Currently this feature is only supported by nodejs and java.
+
+Components created using odo by default run in debug mode (kind of debug mode which doesn't break into debug session unless attached to). This means that a debugger agent is already running on the component side on a specific port. So user just needs to start the port forwarding using 
```

Comment:
In the last cabal i heard that there is a env var which can control the enable and disable of debug mode. Can you please add those information here

Reply:
that should be added in a different PR and hence my suggestion would be to update the doc as part of that PR as well